SQL/database_changelog.txt

```diff
@@ -1,3 +1,13 @@
+1 September 2016, by Jordie0608
+
+Modified table 'notes', adding column 'secret'.
+
+ALTER TABLE `feedback`.`notes` ADD COLUMN `secret` TINYINT(1) NOT NULL DEFAULT '0'  AFTER `server`
```

Comment:
I'd make them default to secret then players can ask admins to review their notes and unhide the unhidables.

It would be much much easier. then trying to go thru and hide all the ones that should be secret.